### DISCUSSION TITLE
Why the name Maddy? What does it mean?

### DISCUSSION THREAD

**Discussion by coolaj86** [emersion/maddy#560] [General]
So Mad about the state of email servers that you had to write one?

Something as simple as Caddy, but for Mail?

What's the etymology and meaning of Maddy?

**Comment by foxcpp** [comment]
I believe @emersion would have a better answer for this. He picked this name initially (as it was his project).
From what I know, he indeed intended it to be "caddy for mail".

**Reply by emersion** [reply]
Right, the reason was mostly that it was intended to be Caddy for mail, and also happened to be a first name. Decided on a whim, don't read too much into it!
